poc/prg.sage

```diff
@@ -5,10 +5,10 @@ from Cryptodome.Cipher import AES
 from Cryptodome.Hash import CMAC, cSHAKE128
 from Cryptodome.Util import Counter
 from Cryptodome.Util.number import bytes_to_long
-from sagelib.common import TEST_VECTOR, VERSION, Bytes, Error, Unsigned, \
-                           format_custom, zeros, from_le_bytes, gen_rand, \
-                           next_power_of_2, print_wrapped_line, to_be_bytes, \
-                           to_le_bytes, xor, concat
+from common import TEST_VECTOR, VERSION, Bytes, Error, Unsigned, \
+                   format_custom, zeros, from_le_bytes, gen_rand, \
+                   next_power_of_2, print_wrapped_line, to_be_bytes, \
+                   to_le_bytes, xor, concat
```

Comment:
FWIW I started moving each imported thing onto its own light, in order to more easily maintain alphabetical order:
```python
from common import \
     TEST_VECTOR \
     VERSION \
...
```

Reply:
I think I'll just run `isort` on this. It'll use parentheses instead of backslash escaping newlines, FWIW.

Reply:
I think I'm going to leave the import statements as-is for now. In the future, maybe we could add `autopep8` and `isort` to a Makefile target, and automate this formatting.

Reply:
Sounds like  plan to me.